circle.yml

```diff
@@ -472,8 +472,9 @@ workflows:
     lint:
         jobs:
             - lint-markdown
-    build-base-images:
+    build-browser-images:
         jobs:
-            - build-base-image:
-                name: "base 16.14.2"
-                dockerTag: "16.14.2"
\ No newline at end of file
+            - build-browser-image:
+                name: "browser node16.14.0-edge"
+                dockerTag: "node16.14.0-edge"
+                edgeVersion: "Microsoft Edge "
```

Comment:
Might want trim this string in the generator so it doesn't have an empty space at the end